arrocke/gloss-translation
# refactor: upgrade lucia and prisma

## What has changed

* Upgraded lucia from v1 to v3 - overall, this simplifies out auth significantly and the code is much easier to understand
  * This upgrade removes the concept of keys - now we just have Users and Sessions
  * Added a table for invitations since keys were handling that before.
  * Added oslo dependency to do password hashing
* Upgraded prisma to v5 - this should bring some performance boosts 



## QA Steps

1. Run `nx run db:prisma migrate dev`
2. Confirm you can still log in with existing accounts
3. Invite a new user to a language
4. Invite a new user to the platform
5. Update email and password for existing user



## Post-Deployment
